Microsoft.Toolkit.Uwp.UI.Controls/DropShadowPanel/DropShadowPanel.cs

```diff
@@ -74,140 +49,24 @@ public DropShadowPanel()
                 _shadowVisual.Shadow = _dropShadow;
             }
 
-            ElementCompositionPreview.SetElementChildVisual(ShadowElement, _shadowVisual);
-        }
-
-        /// <summary>
-        /// Gets or sets the casting element.
-        /// </summary>
-        public FrameworkElement CastingElement
```

Comment:
Need a deprecated CastingElement property. Users could have set the content as such
```
<controls:DropShadowPanel>
    <controls:DropShadowPane.CastingElement>
        <TextBlock />
    </controls:DropShadowPanel.CastingElement>
</controls:DropShadowPanel>
```